### DISCUSSION TITLE
Can't get it to work

### DISCUSSION THREAD

**Discussion by davek00997744** [AntiMicroX/antimicrox#613] [Q&A]
Looks like a very useful app but does not work fully.

I can get it to map my buttons, but it refuses to load the last used profile on restart, it also confuses commands when you have a split yoke like the "velocity flight one" - this appears as two yokes.

I logged a bug report nearly two months back, uploaded logs and nothing has been heard since - is it supported anymore or dead now?

This is on Windows11

**Comment by CabeoC** [comment]
I have this same problem on Garuda Linux where it does not load a configuration, even when opening the .gamecontroller.amgp file in it and when opening the file within the app. It simply does not load a configuration. Anyone know of a fix?

**Reply by pktiuk** [reply]
Just use search and maybe there is something similar https://github.com/AntiMicroX/antimicrox/issues?q=is%3Aissue+config

**Reply by CabeoC** [reply]
Thank you, I see where this is addressed in #618 and #609, and has been fixed in v3.3.2. I am running Garuda Linux, which is currently distributing v3.3.1 at the time of writing. I got the latest AppImage, and the problem is indeed fixed!

**Comment by davek00997744** [comment]
When will v3.3.2 be released please ?

**Reply by pktiuk** [reply]
Today

**Reply by davek00997744** [reply]
Great, thanks
